tools-local/WasmAppBuilderConsole/Program.cs

```diff
@@ -0,0 +1,233 @@
+// -*- indent-tabs-mode: nil -*-
```

Comment:
Hmm, is this an expected part of the copyright header?

Reply:
Removed it.